packages/client-sdk-web/src/utils/web-client-utils.ts

```diff
@@ -20,11 +20,24 @@ export function createCallMetadata(
 export function getWebControlEndpoint(
   credentialProvider: CredentialProvider
 ): string {
-  return `web.${credentialProvider.getControlEndpoint()}`;
+  if (credentialProvider.isControlEndpointOverridden()) {
+    return withProtocolPrefix(credentialProvider.getControlEndpoint());
+  }
+  return withProtocolPrefix(`${credentialProvider.getControlEndpoint()}`);
 }
 
 export function getWebCacheEndpoint(
   credentialProvider: CredentialProvider
 ): string {
-  return `web.${credentialProvider.getCacheEndpoint()}`;
+  if (credentialProvider.isCacheEndpointOverridden()) {
+    return withProtocolPrefix(credentialProvider.getCacheEndpoint());
+  }
+  return withProtocolPrefix(`${credentialProvider.getCacheEndpoint()}`);
```

Comment:
Not sure if I see the differences between line 33 and line 35?

Reply:
good catch, i borked something here, will fix